tmt/base.py

```diff
@@ -456,6 +456,12 @@ def export(self, format_='yaml', keys=None):
             tmt.export.export_to_nitrate(self)
 
         # Common node export otherwise
+        elif keys == 'fmf-id':
+            if format_ == 'dict':
+                return self.fmf_id
+            else:
```

Comment:
Please add explicit `elif format_ == "yaml": ` and raise exception in final `else`.  Otherwise `tmt tests export --identifiers --format BOOO` is accepted.

Reply:
Will do, thank you!